manim/mobject/geometry.py

```diff
@@ -1554,3 +1556,221 @@ def __init__(self, main_shape, *mobjects, **kwargs):
+    all 8 possible angles defined by two lines.
+
+    ..manim:: RightArcAngleExample
+        :save_last_frame:
+
+        class RightArcAngleExample(Scene):
+            def construct(self):
+                line1 = Line( LEFT, RIGHT )
+                line2 = Line( DOWN, UP )
+                rightarcangles = [
+                    ArcAngle(line1, line2, dot=True),
+                    ArcAngle(line1, line2, radius=0.4, quadrant=(1,-1), dot=True, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8, dot=True, dot_color=YELLOW, dot_radius=0.05, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED, dot=True, dot_color=GREEN, dot_radius=0.07),
+                ]
+                line_list = VGroup( *[VGroup() for k in range(4)] )
+                for k in range(4):
+                    linea = line1.copy()
+                    lineb = line2.copy()
+                    line_list[k].add( linea )
+                    line_list[k].add( lineb )
+                    line_list[k].add( rightarcangles[k] )
+                line_list.arrange_in_grid(buff=1.5)
+                self.add(
+                    line_list
+                )
+
+    ..manim:: ArcAngleExample
+        :save_last_frame:
+
+        class ArcAngleExample(Scene):
+            def construct(self):
+                line1 = Line( LEFT + (1/3) * UP, RIGHT + (1/3) * DOWN )
+                line2 = Line( DOWN + (1/3) * RIGHT, UP + (1/3) * LEFT )
+                arcangles = [
+                    ArcAngle(line1, line2),
+                    ArcAngle(line1, line2, radius=0.4, quadrant=(1,-1), other_angle=True),
+                    ArcAngle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED),
+                    ArcAngle(line1, line2, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.4, quadrant=(1,-1)),
+                    ArcAngle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8),
+                    ArcAngle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED, other_angle=True),
+                ]
+                line_list = VGroup( *[VGroup() for k in range(8)] )
+                for k in range(8):
+                    linea = line1.copy()
+                    lineb = line2.copy()
+                    line_list[k].add( linea )
+                    line_list[k].add( lineb )
+                    line_list[k].add( arcangles[k] )
+                line_list.arrange_in_grid(n_rows=2, n_cols=4, buff=1.5)
+                self.add(
+                    line_list
+                )
+
+    """
+
+    def __init__(
+        self,
+        line1,
+        line2,
+        radius=0.3,
+        quadrant=(1, 1),
+        other_angle=False,
+        dot=False,
+        dot_radius=0.03,
+        dot_distance=1.8,
+        dot_color=WHITE,
```

Comment:
Good job in setting meaningful defaults - quadrant especially.

Reply:
Cheers! I changed `dot_radius`to `None` and the default will be calculated as 1/10 of the arc radius. Those two sizes should be related most of the time.